NavigationReactNative/sample/twitter/App.js

```diff
@@ -24,29 +24,40 @@ timeline.renderScene = ({id}) => <Timeline timeline={getTimeline(id)}  />;
 var notificationsNavigator = new StateNavigator(stateNavigator);
 stateNavigator.navigate('home');
 
-export default () => (
-  Platform.OS === 'ios' ? (
-    <TabBarIOS>
-      <TabBarItemIOS title="Home">
-        <NavigationHandler stateNavigator={stateNavigator}>
-          <NavigationStack />
-        </NavigationHandler>
-      </TabBarItemIOS>
-      <TabBarItemIOS title="Notifications" onPress={() => {
-        if (!notificationsNavigator.stateContext.state)
-          notificationsNavigator.navigate('notifications');
-      }}>
-        <NavigationHandler stateNavigator={notificationsNavigator}>
-          <NavigationStack />
-        </NavigationHandler>
-      </TabBarItemIOS>
-    </TabBarIOS>
-  ) : (
-    <NavigationHandler stateNavigator={stateNavigator}>
-      <NavigationStack
-        crumbStyle={from => from ? 'scale_in' : 'scale_out'}
-        unmountStyle={from => from ? 'slide_in' : 'slide_out'}>
-      </NavigationStack>
-    </NavigationHandler>
-  )
-);
+export default() => {
+  const [isNotified,
```

Comment:
Good name choice. Maybe we can go one better and change it to just "notified"?

Reply:
Ok. Will make the change.